.devcontainer/devcontainer.json

```diff
@@ -60,7 +38,6 @@
 				"mikestead.dotenv",
 				"ms-azuretools.vscode-docker",
 				"ms-python.python",
-				"ms-python.isort",
```

Comment:
Have you also checked if it is still best practice to use "ms-python.python" along with "ms-python.vscode-pylance" for general python language support?

Reply:
Ah, no it looks like pylance gets automatically installed with the ms python extension now, so no need to independently specify it.

Reply:
OK, perfect.